mlflow/ml-package-versions.yml

```diff
@@ -44,7 +44,9 @@ pytorch-lightning:
     maximum: "1.6.0"
     requirements:
       "< 1.2.0": ["torch<1.11.0", "torchvision", "scikit-learn"]
-      ">= 1.2.0": ["torchvision", "scikit-learn"]
+      ">= 1.2.0, < 1.3.0": ["torchvision", "scikit-learn"]
+      ">= 1.3.0, < 1.5.0": ["torchmetrics<0.8.0", "torchvision", "scikit-learn"]
```

Comment:
Can we add an inline comment with a link to the torchmetrics 0.8.0 issue description for future reference?

Reply:
Definitely!